runtime/dynstats.c

```diff
@@ -468,6 +566,10 @@ dynstats_findBucket(const uchar* name) {
 			}
 			b = b->next;
 		}
+
+		if (b && b->persistStateInterval) {
+			dynstats_openStateFile(b);
+		}
```

Comment:
This does not belong here. I think you want to put this in `dynstats_newBucket`. Loading it here would artificially bump the counters `n - 1` times, where `n` is the number of `dyn_inc` call-site initialization(s). So if counter `foo` has value `10` and we have 3 sites, we'd end up with value 30 (instead of 10).